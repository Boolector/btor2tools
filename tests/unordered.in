2 zero 2
1 zero 1
