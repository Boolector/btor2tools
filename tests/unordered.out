*** catbtor: parse error in 'unordered.in' line 2: id out-of-order
